pkg/aws/build.go

```diff
@@ -6,10 +6,10 @@ import "fmt"
 // for an invocation image using this mixin
 func (m *Mixin) Build() error {
 	// TODO: This gets whatever the latest version of the cli is, there isn't a way for us to say what version we are using
-	fmt.Fprintln(m.Out, `RUN apt-get update && apt-get install -y curl unzip python less groff`)
-	fmt.Fprintln(m.Out, `RUN curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "/tmp/awscli-bundle.zip"`)
-	fmt.Fprintln(m.Out, `RUN unzip /tmp/awscli-bundle.zip -d /tmp`)
-	fmt.Fprintln(m.Out, `RUN /tmp/awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws`)
-	fmt.Fprintln(m.Out, `RUN rm -fr /tmp/awscli-bundle.zip /tmp/awscli-bundle`)
+	fmt.Fprintln(m.Out, `RUN apt-get update && apt-get install -y curl unzip glibc less groff`)
```

Comment:
I tested out your change locally by running `make build install` from the aws mixin directory, and then going into a bundle directory that uses the aws mixin and then running `porter build`.

It ran into an error trying to install glibc, because I guess glibc is distributed as the libc6 package on debian (which is the distro that the mixins use). I have also included a flag so that apt only installs what we ask for and not any recommended packages to keep the image size down.

```suggestion
	fmt.Fprintln(m.Out, `RUN apt-get update && apt-get install -y --no-install-recommends curl unzip libc6 less groff`)
```

Reply:
thanks @carolynvs

Reply:
@carolynvs I pushed the fix for this now